data/templated/example.xml

```diff
@@ -79,6 +79,7 @@
          <value>255.0</value>
       </values>
    </template>
+   <!-- FIXME: This results in primary key failures due to duplicate keys.
```

Comment:
Temporarily removes a faulty INSERT test query from the templated benchmark example introduced in #386 since it results in duplicate keys.